### DISCUSSION TITLE
Removed from F-Droid?

### DISCUSSION THREAD

**Discussion by shivagnihotri** [SuhasDissa/Food-E-App#52] [General]
Greetings from Germany! Why is Food-E not available in F-Droid anymore? Would be wonderful if you re-add it there.

**Comment by SuhasDissa** [comment]
Hey there! 
I think there's some misunderstanding here.

I haven't released any apps on F-Droid yet, but I'm thinking about it! 
It would be great to have this app available on F-Droid. I'll keep you updated on my progress.

Thanks for the suggestion! 😊

**Comment by shivagnihotri** [comment]
Thank you very much! Strange thing, I had installed your app from F-Droid and wondered why I did not reveive updates from there, until I noticed that the app is not there anymore... 🥸

**Comment by SuhasDissa** [comment]
I created a [merge request](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/13221) at F-Droid yesterday, and I'm still waiting for them to approve it. It's starting to feel like it's taking longer than I expected.
I'm really excited about this, and I hope they approve it! Fingers crossed!

**Comment by CEOXeon** [comment]
> Thank you very much! Strange thing, I had installed your app from F-Droid and wondered why I did not reveive updates from there, until I noticed that the app is not there anymore... 🥸

For me it was in the IzzySoft Repo which I added to the F Droid repo list in the app.

**Comment by shuvashish76** [comment]
Live on F-Droid: https://f-droid.org/en/packages/app.suhasdissa.foode/ 🎉
Feel free to use their badge: https://gitlab.com/fdroid/artwork/-/blob/master/badge/README.md

**Reply by SuhasDissa** [reply]
Thanks for letting me know :)
